README.md

```diff
@@ -2,7 +2,70 @@
 
 A tool to migrate API usage to Dart 2.
 
-## Features and bugs
+### What does it do?
+
+`dart2_fix` migrates API usage from Dart 1 code to Dart 2 code. Currently, it focuses on updating
+deprecated constant names (like `dart:convert`'s `UTF8` to `utf8`).
+
+### How do I use it?
+
+To install, run `pub global activate dart2_fix`. Then, from your project directory, run:
+
+`pub global run dart2_fix`
```

Comment:
...I think `pub global activate` will give you an FYI if you path isn't set. 

So `dart2_fix` will "just work" right?

Reply:
Yeah, I have it more explicit here to make this more error-proof. I'll land as is, but happy to have this updated if you feel we should.